pages/dkp/kommander/2.2/DKP-insight/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Insight
+title: Insight
+excerpt: Use DKP insight to monitor and tune your enviornment
+menuWeight: 55
+beta: true
+enterprise: false
+---
+
+<p class="message--note"><strong>NOTE:</strong> Insights capability is offered with this release as a Technical preview only. Future enhancements include add-on licensing, a seamless user experience, and additional insight items.</p>
+
+DKP Insights is D2iQ’s latest analytics tool, designed to assist Kubernetes Administrators with daily tasks, such as making sure clusters are secure, debugging, monitoring, and anticipating future issues with your cluster. Insights provides you predictive analytics capability that provides greater support productivity, speed, and reduced cost. An Insights module, integrated into the DKP, collects configuration, logs, events, and metrics from DKP deployments and generates predictive insights on potential issues of varying critical levels so that you can quickly identify and resolve issues that would save a lot of time and money by not escalating to D2IQ's support.
+
+Insights sits inside of DKP within your environment, so it works seamlessly within networked or air-gapped environments. Insights collects data from multiple points, such as metrics, logs, and events, and then uses a set of heuristics and models to interpret the data and let you know what’s going on. Events display in a user-friendly, interactive table, showing what’s critical, warnings, and general notices that you need to be aware of.
+
+Your Infrastructure Ops. Manager can filter the insight summary cards by one or many insight categories for a selected cluster and project:
```

Comment:
"Infrastructure Ops. Manager" is a title that needs introducing somewhere...   I'm not aware of any other places in our documentation where we suggest or document specific 'user roles' such as 'Infrastructure Ops. Manager'.  

```suggestion
You can filter the Insights summary cards using one or more categories for a selected cluster and project:
```